pkgs/os-specific/linux/kernel/common-config.nix

```
@@ -1023,6 +1023,8 @@ let
       CROS_EC_ISHTP = module;
 
       CROS_KBD_LED_BACKLIGHT = module;
+
+      TCG_TIS_SPI_CR50 = yes;
```

```suggestion
      TCG_TIS_SPI_CR50 = whenAtLeast "5.5" yes;
```
https://cateee.net/lkddb/web-lkddb/TCG_TIS_SPI_CR50.html

---

Thank you!